tasks/cssmin.js

```diff
@@ -31,6 +31,20 @@ module.exports = function (grunt) {
 
       try {
         compiled = new CleanCSS(options).minify(availableFiles);
+
+        // something went horribly wrong
+        if (compiled.errors.length) {
+          grunt.warn(compiled.errors.toString());
```

Comment:
`grunt.warn` is an alias for `grunt.fail.warn` http://gruntjs.com/api/grunt.fail